src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1750,6 +1751,76 @@ public static void ConsumeLotsOfComments(string valueString, bool insideArray, s
             Assert.Equal(dataUtf8.Length, json.BytesConsumed);
         }
 
+        [Theory]
+        [MemberData(nameof(SingleLineCommentData))]
+        public static void ConsumeSingleLineCommentSingleSpanTest(string[] inputData)
+        {
+            string expected = inputData[inputData.Length - 1];
+            inputData = inputData.Take(inputData.Length - 1).ToArray();
+
+            var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });
+            int bytesConsumed = 0;
+            var dataUtf8 = new byte[] { };
+
+            for (int count = 0; count < inputData.Length; ++count)
+            {
+                bool isFinal = (count == inputData.Length - 1);
+                dataUtf8 = dataUtf8.Concat(Encoding.ASCII.GetBytes(inputData[count])).ToArray();
+                var json = new Utf8JsonReader(dataUtf8, isFinalBlock: isFinal, state);
+                while (json.Read())
+                {
+                    switch (json.TokenType)
+                    {
+                        case JsonTokenType.StartObject:
+                        case JsonTokenType.EndObject:
+                            break;
+                        case JsonTokenType.Comment:
+                            Assert.Equal(Encoding.ASCII.GetBytes(expected), json.ValueSpan.ToArray());
+                            break;
+                        default:
+                            Assert.True(false);
+                            break;
+                    }
+                }
+                dataUtf8 = dataUtf8.Skip(bytesConsumed).ToArray();
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(SingleLineCommentData))]
+        public static void SkipSingleLineCommentSingleSpanTest(string[] inputData)
+        {
+            string expected = inputData[inputData.Length - 1];
+            inputData = inputData.Take(inputData.Length - 1).ToArray();
+
+            var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });
```

Comment:
Should this be `Skip` where `json.TokenType` never == `JsonTokenType.Comment`? Otherwise, the `SkipSingleLineCommentSingleSpanTest` and `ConsumeSingleLineCommentSingleSpanTest` are identical.

Reply:
oops.